dti-website/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "start": "next",
-    "build": "next build && next export",
+    "build": "yarn run pull-from-idol && next build && next export",
```

Comment:
Not sure whether we want this kind of automatic deployment without approval process. Imaging someone added some bad self description or we messed up the json, then it will appear on the website live without any human intervention.